hack/tests/subcommand-scorecard.sh

```diff
@@ -65,44 +79,58 @@ then
 	exit 1
 fi
 
-# test to see if version in config file allows v1alpha1 to be specified
+header_text 'scorecard test to see if version in config file allows v1alpha2 to be specified'
 commandoutput="$(operator-sdk scorecard --config "$CONFIG_PATH_V1ALPHA1" 2>&1)"
-echo $commandoutput | grep "Total Score: 67%"
+echo $commandoutput | grep "Total Score: 64%"
 
+header_text 'scorecard test to see if total score matches expected value'
 commandoutput="$(operator-sdk scorecard --version v1alpha1 --config "$CONFIG_PATH" 2>&1)"
-echo $commandoutput | grep "Total Score: 67%"
+echo $commandoutput | grep "Total Score: 64%"
 
```

Comment:
Here the test was changed but it is using v1alpha1. Is it correct? Should this change affect the v1alpha1 as well?

Reply:
yes, because the alm-examples test is removed.

Reply:
Why we are removing it here?
If we will remove it, then I think that it should be another PR and also it needs to have an entry in the CHANGELOG.

Reply:
explained in another comment. The test still, it will just made in the API lib now.